scapy/contrib/automotive/someip.py

```diff
@@ -215,9 +215,12 @@ def fragment(self, fragsize=1392):
 
 
 def _bind_someip_layers():
-    for i in range(15):
-        bind_layers(UDP, SOMEIP, sport=30490 + i)
-        bind_layers(TCP, SOMEIP, sport=30490 + i)
+    bind_top_down(UDP, SOMEIP, sport=30490, dport=30490)
 
+    for i in range(15):
+        bind_bottom_up(UDP, SOMEIP, sport=30490 + i)
+        bind_bottom_up(TCP, SOMEIP, sport=30490 + i)
+        bind_bottom_up(UDP, SOMEIP, dport=30490 + i)
+        bind_bottom_up(TCP, SOMEIP, dport=30490 + i)
 
 _bind_someip_layers()
```

Comment:
Why is this a function BTW ?

Reply:
no idea, I'm not the original author

Reply:
Never mind I found my answer.
All good !
https://github.com/secdev/scapy/pull/1720#discussion_r238103528